server/config/constants.go

```diff
@@ -53,16 +53,31 @@ const (
 	PropUserAgent = "user_agent_type"
 )
 
-// KV namespace. The use of '.' in the prefixes is to avoid conflicts with
-// base64 URL encoding that already uses '-' and '_'.
+// KV namespace
+//
+// Keys starting with a '.' are reserved for app-specific keys in the "hashkey"
+// format. Hashkeys have the following format (see service_test.go#TestHashkey
+// for examples):
+//
+//  - global prefix of ".X" where X is exactly 1 byte (2 bytes)
+//  - bot user ID (26 bytes)
+//  - app-specific prefix, limited to 2 non-space ASCII characters, right-filled
```

Comment:
Out of scope of this PR: What is the use case of such prefix? KVList per prefix?

Reply:
`KVList` would be costly as there is no parameter for prefix. We would have to use SQL if we wanted to efficiently loop through prefixed keys.

Reply:
Good point @mickmister :+1: